src/xrc/src/utils.rs

```diff
@@ -69,7 +96,8 @@ pub(crate) fn standard_deviation(rates: &[u64]) -> u64 {
         .map(|rate| ((*rate as i128).saturating_sub(mean)).saturating_pow(2) as u128)
         .sum::<u128>()
         .saturating_div(count.saturating_sub(1) as u128);
-    (variance as f64).sqrt() as u64
+    println!("Variance: {}", variance);
```

Comment:
Can this be removed?

Reply:
Oops. Yes, it can. I'll remove it. Thanks!

Reply:
Done.